README.md

```diff
@@ -1 +1,20 @@
 # S-ENDA-Prototype
+## Getting started
+Start all prototype components.
+
+First install dependencies by following the documentation here:
```

Comment:
Also, since vagrant documentation setup has been merged, we should refer to those docs for how to install dependencies:
https://s-enda-documentation.readthedocs.io/en/latest/devel_environ.html#Installation

Reply:
I will just update myself, and then do the merge  :)